fastlane/lib/fastlane/actions/latest_testflight_build_number.rb

```diff
@@ -72,7 +78,7 @@ def self.available_options
                                        description: "The version number whose latest build number we want",
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :initial_build_number,
-                                       env_name: "INTITIAL_BUILD_NUMBER",
+                                       env_name: "INITIAL_BUILD_NUMBER",
```

Comment:
What I mean, please do the following
```ruby
           FastlaneCore::ConfigItem.new(key: :initial_build_number,
                                        env_name: "INITIAL_BUILD_NUMBER",
                                         default_value: ENV["INTITIAL_BUILD_NUMBER"]
                                         ....
```
This way, it will support both parameters, so we won't break any existing setups from our users 👍